Components/Modal/modal.css

```diff
@@ -0,0 +1,79 @@
+/* ---------------------------------> Simple Modal <------------------------------------- */
+.modal {
+    position: fixed;
+    top: 0;
+    left: 50%;
+    z-index: 1000;
```

Comment:
Such values for z-index do not reflect well. Study your code thoroughly, analyze it and assign an accurate z-index value here.